usagov/vote-gov-nvrf-app
# VOTE-3070 touchpoints form spacing

Jira ticket:https://cm-jira.usa.gov/browse/VOTE-3070

Test steps:
1. Fill out the form in English and view the last page (delivery)
2. Confirm there is a new heading and spacing for the touchpoints form
![image](https://github.com/user-attachments/assets/70658e6a-4ef9-45cc-b943-d673a95a33ea)
